src/test/java/uk/gov/companieshouse/web/accounts/controller/smallfull/StepsToCompleteControllerTests.java

```diff
@@ -212,4 +220,35 @@ void postRequestCompanyAccountsServiceCreateSmallFullAccountsFailure() throws Ex
                 .andExpect(status().isOk())
                 .andExpect(view().name(ERROR_VIEW));
     }
+
+    @Test
+    @DisplayName("Post steps to complete failure path for create statements resource")
+    void postRequestStatementsServiceCreateStatementsFailure() throws Exception {
+
+        when(transactionService.createTransaction(COMPANY_NUMBER)).thenReturn(TRANSACTION_ID);
+
+        LocalDate periodEndOn = LocalDate.now();
+
+        NextAccountsApi nextAccounts = new NextAccountsApi();
+        nextAccounts.setPeriodEndOn(periodEndOn);
```

Comment:
Could we remove the local `periodEndOn` variable above in favour of calling `.setPeriodEndOn(LocalDate.now())`?